utils/src/test/java/by/khadasevich/utils/StringUtilsTest.java

```diff
@@ -0,0 +1,22 @@
+package by.khadasevich.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StringUtilsTest {
+
+    @Test
+    void isPositiveNumber() {
+        assertTrue(StringUtils.isPositiveNumber("1f"));
```

Comment:
each test must be independent, so here you have to create a test for every test case or use parametrized tests since you are using junit5